src/driver/odoc_driver.ml

```diff
@@ -205,6 +211,7 @@ let run mode
     else []
   in
   Logs.debug (fun m -> m "XXXX Remaps length: %d" (List.length remaps));
+  Format.eprintf "Starting the compilation process... \n%!";
```

Comment:
Shouldn't this be `Logs.info` ?

Reply:
Good point. However, shouldn't this be `Logs.app`?

Currently, the `Info` level is a bit too verbose. Eg:

```
odoc_driver.exe: [INFO] Unable to determine library of archive ocamlmiddleend: Ignoring.
odoc_driver.exe: [INFO] No files for module: Tophooks
odoc_driver.exe: [INFO] Unable to determine library of archive odoc_info: Ignoring.
odoc_driver.exe: [INFO] No files for module: Findlib
odoc_driver.exe: [INFO] No files for module: Findlib_config
odoc_driver.exe: [INFO] No files for module: Fl_args
odoc_driver.exe: [INFO] No files for module: Fl_lint
odoc_driver.exe: [INFO] No files for module: Fl_meta
odoc_driver.exe: [INFO] No files for module: Fl_metascanner
odoc_driver.exe: [INFO] No files for module: Fl_metatoken
odoc_driver.exe: [INFO] No files for module: Fl_package_base
odoc_driver.exe: [INFO] No files for module: Fl_split
odoc_driver.exe: [INFO] No files for module: Fl_topo
odoc_driver.exe: [INFO] No files for module: Ocaml_args
odoc_driver.exe: [INFO] No files for module: Fl_dynload
odoc_driver.exe: [INFO] No files for module: Topfind
```
will be displayed for any invokation of `odoc_driver`. I believe this should be fixed in another PR!

Reply:
Yes, that's `Logs.app` !